### PR TITLE
Add French translation file

### PR DESCRIPTION
**Pull request by HLFH**
Add Isso support, so Ghost users will be able to use either Disqus (proprietary) or Isso (open source). Using a commenting system like Isso will be much faster to enable after a `ghost update`.

### CODE REVIEW

**Comment by HLFH** [comment]
EDIT: traduction for French has been added.

**Comment by HLFH** [comment]
EDIT2: I added a fix for feedly URLs. Related to https://github.com/TryGhost/Casper/pull/418

**Comment by NerosTie** [comment]
screenshot_20180126_172000

You need to change this line in `fr.js`:
```"Subscribe to": "S'abonner à",```
to  
```"Subscribe to {blogtitle}": "S'abonner à {blogtitle}",```

(also, it should be `votreemail@exemple.fr`, not `example`)

File: locales/fr.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "Back": "Retour",
+    "Newer Posts": "Articles plus récents",
+    "of": "sur",
```
Comment (juan-g) [inline locales/fr.json]: Because of the recent update, the `"of"` line should be removed.

File: locales/fr.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "Older Posts": "Articles plus anciens",
+    "Page": "Page",
```
Comment (juan-g) [inline locales/fr.json]: `"Page"` should be `"Page {page} of {pages}"` now.

File: locales/fr.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "Subscribe": "S'abonner",
+    "Subscribe to": "S'abonner à",
```
Comment (juan-g) [inline locales/fr.json]: `"Subscribe to"` should be `"Subscribe to {blogtitle}"`.

File: locales/fr.json [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    "Get the latest posts delivered right to your inbox": "Recevez les articles récents directement dans votre boîte de réception",
+    "Latest Posts": "Articles récents",
+    "No posts": "Aucun article",
+    "Read More": "En savoir plus",
+    "Read <a href='{url}'>more posts</a> by this author": "Lire <a href='{url}'>plus d'articles</a> de cet auteur",
+    "See all % posts": "Voir ces % articles",
+    "Share this": "Partager",
+    "Stay up to date! Get all the latest & greatest posts delivered straight to your inbox": "Soyez au goût du jour ! Recevez tous les articles récents et sympas directement dans votre boîte de réception",
+    "Subscribe": "S'abonner",
+    "Subscribe to": "S'abonner à",
```
Comment (juan-g) [inline locales/fr.json]: `"Subscribe"` and `"Subscribe to"` were duplicate lines that should be removed.

**Comment by juan-g** [comment]
Thank you, and sorry for not adding before the [CONTRIBUTING.md](https://github.com/juan-g/WorldCasper2/blob/i18n-translatable-frontend/CONTRIBUTING.md) file. Basically, to maintain compatibility between Casper and WorldCasper2:

> Thank you for any contribution to theme translations, mainly new _.json_ files (based on the English default _en.json_) to translate more languages. (...)
> 
> For contributions not related to translations, please submit pull requests to [Casper](https://github.com/TryGhost/Casper), (...) Contributions that are accepted into Casper will appear in WorldCasper2 as well with later updates.

For example, the fix for Feedly URLs was added to Casper (upstream of WorldCasper2), and is therefore now in WorldCasper2 as well with the recent update.

Your file _fr.json_ would be indeed useful for many people if you can please prepare a pull request for just your French translations file, with the above mentioned updates. Thank you for your kindness.

**Comment by HLFH** [comment]
Ok. I'll do.

**Comment by duvrai** [comment]
I updated the french file for the latest version. I hope it's ok to post it like this:
[fr.json](https://github.com/juan-g/WorldCasper2/files/1863594/fr.json.txt)

**Comment by juan-g** [comment]
Thanks, @duvrai.

@HLFH Would it be OK if I add commits to this pull request by pushing to the _i18n-translatable-frontend_ branch on your _HLFH/WorldCasper2_ repository?

If yes, you can have _"Allow edits from maintainers"_ checked on this pull request. Or, if you prefer, I can add commits directly to my branch instead of this pull request. Thank you!

**Comment by juan-g** [comment]
Just merged two commits by @HLFH and @duvrai adding and updating the French translation file. This has been done directly, so we can close this pull request. Thank you!

**Pull request closed without merging**